src/qt/intro.cpp

```diff
@@ -23,7 +23,7 @@
 
 static const uint64_t GB_BYTES = 1000000000LL;
 /* Minimum free space (in GB) needed for data directory */
-static const uint64_t BLOCK_CHAIN_SIZE = 120;
+static const uint64_t BLOCK_CHAIN_SIZE = 50;
```

Comment:
```suggestion
static const uint64_t BLOCK_CHAIN_SIZE = 55;
```
Lets have a bit more space just to be sure

Reply:
It uses 47 GB on my node, but I don't have txindex.